NFT/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[project]
+name = "NFT"
+version = "0.0.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+license = "Apache-2.0"
+
+[dependencies]
+fuels = "0.16"
+fuels-abigen-macro = "0.16"
```

Comment:
I was under the impression `fuels-abigen-macro` was no longer needed as a separate import as of `0.16`: https://github.com/FuelLabs/fuels-rs/pull/393

Reply:
Correct